src/devices/vdml_motors.c

```diff
@@ -275,9 +270,10 @@ int32_t motor_set_pos_pid(uint8_t port, const motor_pid_s_t pid) {
 	out.kd = pid.kd;
 	set_pos_pid(port - 1, out);
 	vexDeviceMotorPositionPidSet(device->device_info, &out);
-	return_port(port - 1, 1);
```

Comment:
I feel like either way, it would stay consistent to clean out these methods, because not only is the code useless, it might actually be doing something unintended.

Reply:
agreed... I was also told by VEX that there are some weird behaviors with these functions (such as trying to set the value equal to 0 will effectively do nothing to the current value) so having the functionality gutted would arguably make the behavior of these functions more consistent lol